docs/rest-apis.md

```diff
@@ -45,115 +49,218 @@ Let's create a `GET` request for the `/hello` route.
 
+:::tip
+`PUT`, `PATCH`, and `DELETE` requests work the same way. Just add a file with the folder name to the route directory.
+:::
+
 ### API Dynamic Paths
 
 In cases where you would like to have a dynamic path just wrap your folder name in `{}`.
 
-Let's add a `/hello/{name}` route to the Demo API.
-
+#### GET /hello/\{name\}
+Let's create a `GET /hello/{name}` and say *"Hello"* to the `name` provided in the route path:
 ```bash
 mkdir less/apis/demo/hello/{name}
 ```
 
-We now have a Chat API with 2 routes:
+<Tabs groupId="programming-language" queryString="programming-language">
+
+  <TabItem value="nodejs" label="Node.js">
+  ```bash
+  touch less/apis/demo/hello/{name}/get.js
+  ```
+
+  Here's how we can access the value of `name` from the route path:
+
+  ```js {2} title="less/apis/demo/hello/{name}/get.js" showLineNumbers
+  exports.process = async (request, response) => {
+    const { name } = request.params; 
+    response.body = 'Hello, ' + name + '.';
+    response.statusCode = 200;
+    return response;
+  };
+  ```
+  </TabItem>
+  
+</Tabs>
+
+:::note
+We now have a Demo API with 3 routes:
 - `GET /hello`
+- `POST /hello`
 - `GET /hello/{name}`
+:::
+
+## HTTP Request/Response Properties
 
-#### Access Dynamic Path Values
-You can access the value of the dynamic path through the `request` object.
+### Status Code (required)
+You can set the response HTTP status code through the `statusCode` property.
 
-Here's how we can access the value of `name`:
 <Tabs groupId="programming-language" queryString="programming-language">
   
   <TabItem value="nodejs" label="Node.js">
-  ```js {2} title="less/apis/demo/hello/{name}/get.js" showLineNumbers
-  module.exports.process = async (request, response) => {
-      const { name } = request.params; 
+  ```js {2} title="less/apis/demo/hello/get.js" showLineNumbers
+  exports.process = async (request, response) => {
+    response.statusCode = 200;
+    return response;
   };
   ```
   </TabItem>
 
-  <TabItem value="py" label="Python">
-  ```py {2} title="less/apis/demo/hello/{name}/get.py" showLineNumbers
-  def process(request, response):
-      name = request['params']['name']
+</Tabs>
+
+### Headers
+You can access request and response headers through the `headers` object.
+
+Here's how we can access the *Accept* header in a request and set the *Content-Type* in a response:
+<Tabs groupId="programming-language" queryString="programming-language">
+  
+  <TabItem value="nodejs" label="Node.js">
+  ```js {2,5} title="less/apis/demo/hello/get.js" showLineNumbers
+  exports.process = async (request, response) => {
+    const accept_header = request.headers.Accept;
+    if (accept_header === 'application/json') {
+      response.body = { hello: 'world' };
+      response.headers['Content-Type'] = accept_header;
+    }
+    else {
+      response.body = 'Hello, world.';
+    }
+    response.statusCode = 200;
+    return response;
+  };
   ```
   </TabItem>
   
 </Tabs>
 
-## HTTP Request
+
```

Comment:
@nnascim we can add example of how to deal with multiple values for a query paramter

Reply:
I created [this ticket](https://github.com/chuva-io/less-site/issues/27) for it.